R/sulpher.R

```diff
@@ -71,11 +71,11 @@ calc_slv <- function(B_LU_BRP, B_SOILTYPE_AGR, B_AER_CBS,A_SOM_LOI,A_S_RT, D_BDS
   
   # Calculate SLV for grass (deze formule: Stot = g/kg en dichtheid in g/cm3)
   dt.grass <- dt[crop_category == "grasland"]
-  dt.grass[, value := 17.8 * A_S_RT * 0.001 * D_BDS * 0.001]
+  dt.grass[, value := 17.8 * A_S_RT * D_BDS * 0.001]
   
   # Calculate SLV for maize for 0-25 cm depth
   dt.maize <- dt[crop_category == "mais"]
-  dt.maize[, value := 41.2 * A_S_RT * 0.001 * D_BDS * 2.5 * 0.001]
+  dt.maize[, value := 41.2 * A_S_RT * D_BDS * 2.5 * 0.001]
   # correction for the length of growing season (43.5%) 
```

Comment:
here the same. if S is too low, there shoulde be another reason than this unit correction